packages/shared/src/schemas.ts

```diff
@@ -62,10 +62,17 @@ export type SwapFee = {
   amount: string;
 };
 
+export type PriceWarning = {
+  threshold: number;
+  warn: boolean;
+};
+
 export type QuoteQueryResponse = {
   intermediateAmount?: string;
   egressAmount: string;
   includedFees: SwapFee[];
+  lowLiquidityWarning: PriceWarning;
+  priceImpactWarning: PriceWarning;
```

Comment:
we need to think about these names to prevent people from missunderstanding them. eg `price impact` is a term that is used by uniswap for something else 🙈  i guess we should as explicit at possible

Reply:
maybe `feeRatioWarning`/`highFeeRatioWarning` if the fees are a substantial amount of the ingress value?

Reply:
`lowLiquidityWarning` is a bit misleading too because this would also happen if we have enough liquidity (but at a bad price). maybe `lowSwapRateWarning`?

Reply:
the other possibility would be not exposing a warning but the actual value that triggers the warning - then the consumer should decide about the thresholds. in this case we could call it:
- `estimatedFeeRatio`
- `estimatedPriceDelta`